README.md

```diff
@@ -504,7 +504,8 @@ the query engine will lookback to normalize the value for any given instant.
 can yield to queries with unintended effects. Lets consider the expression `rate(foo[5m])`. The result
 of this query would have completely different meanings with different step values. Since step parameter
 typically is not controlled by user (as in Grafana), it can have silent negative effects. For example, if
-lookback becomes smaller than step because longer time range was chosen, then query silently drops samples.
+user expands the time range and hardcoded lookback becomes smaller than step, the same query will no
+longer do what the user orginally intends to do because samples wont be included in the query.
```

Comment:
Suggestion: replace
     the same query will no
 longer do what the user orginally intends to do because samples wont be included in the query

with

   the query lookback may not include enough samples for an effective query